src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -1260,7 +1239,7 @@
           }
         }
       },
-      "required": [ "tool" ]
+      "required": [ "results", "tool" ]
```

Comment:
Sorry!!! Let's remove `result` from this list. My bad. I need to talk more to Michael about this.